alembic/operations/base.py

```diff
@@ -127,7 +127,6 @@ def %(name)s%(args)s:
                     "args": args,
                     "apply_kw": apply_kw,
                     "doc": fn.__doc__,
-                    "meth": fn.__name__,
```

Comment:
I'm not sure this is needed for this pr

Reply:
OK, I've added this because pep8 check failed: https://jenkins.sqlalchemy.org/job/pep8/3042/console

Reply:
See 8996e5a.

Reply:
you are correct. It seems to be flake8 3.8.1 that can now detect this.

Sorry I made you revert it. It seems that we need to remove it

Reply:
See 8fca973.

Reply:
oh OK, didnt see that

Reply:
**Federico Caselli** (CaselIT) wrote:

the problem there is that flake8 updated to version 3.8.1 and that like is now flagged as an error

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/alembic/+/1970